plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -263,6 +265,23 @@ jQuery( function( $ ) {
 				$( this ).trigger( 'load' );
 			}
 		} );
+
+		// Custom keyboard navigation that applies only when the slider container is focused.
```

Comment:
I’m happy to merge this, as it appears to follow the correct accessibility behavior.
However, do you think there’s a risk of breaking something?
Some extensions might rely on the current implementation. For example, here’s what happens when I run this code:


```js
const event = jQuery.Event("keyup", {
  key: "ArrowRight",
  code: "ArrowRight",
  keyCode: 39,
  which: 39,
});



jQuery("body").trigger(event);
```


| Before | After |
|--------|--------|
| <video src="https://github.com/user-attachments/assets/1b0a1a0a-dadb-467f-bf68-9dbe96148301" /> |<video src="https://github.com/user-attachments/assets/36d8c9b1-b8b1-4533-b6d1-b6d84b1a06c0" /> |

Reply:
Yeah I think it's OK to leave the old behaviour as it is, as the gallery is being replaced with the new block. I can't reproduce the bug on the new Product Gallery block as it's caused by FlexSlider, which the new gallery isn't using. 

Perhaps we could still add the `tabindex` changes to the Add to Cart With Options block, and remove the FlexSlider changes from this PR?

Reply:
> Perhaps we could still add the tabindex changes to the Add to Cart With Options block, and remove the FlexSlider changes from this PR?

I'm fine with this! If we get some reports, we can investigate on other approaches or merge this PR. What are your thoughts on opening a dedicated PR for `tabindex` patch? In this way, we can keep this code easily accessible if needed in the future.

Reply:
Feel free to add me as a reviewer!